certbot/plugins/selection.py

```diff
@@ -39,6 +39,35 @@ def pick_authenticator(
     return pick_plugin(
         config, default, plugins, question, (interfaces.IAuthenticator,))
 
+def get_installer(config, plugins):
```

Comment:
nit: Can we make this name a little more descriptive? Maybe by adding "unprepared"? Or maybe "required" since this unlike many of the other functions, this should never return `None`.

We have all these `{pick,get,choose]_*` functions in this file and it's not clear until reading the docstrings which does what.

Reply:
I changed the name to `get_unprepared_installer()`